package exceptionhandling;

public class ExceptionExample {
	public static void main(String[] args) {
		try {
			int result = 10 / 0; // This line will throw ArithmeticException
			System.out.println("Result :" + result);
		} catch (ArithmeticException e) {
			System.out.println(e.getMessage()); // This will print the exception message
		}
		System.out.println("Program continues"); // This will always execute
	}
}
